packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -16,6 +16,17 @@ const pgDockerCompose = () => {
   return readFileSync(`${dockerFilesPath}/pg-docker-compose.yml`, 'utf8')
 }
 
+const generationConfig = `{
```

Comment:
Are we providing user the option to pass config through the `config.json` file and not use directives? Because we already have default config in the core.

Reply:
We still need to use directives, but you can imagine schema looking very very verbose if we stat to disable every type. We need a global config for generation and then individual types

Reply:
So let's have directives for applying stuff and users can disable stuff in the config

Reply:
This seems good as config, and we can include pagination in this too if we want later. But what should be the directives we provide now `@sub(new, update) @paginate` etc

Reply:
Yes.. in fact if you look into TODO config already have it so I will add it here as well